website/versioned_docs/version-5.x/module-types.md

```diff
@@ -10,13 +10,13 @@ Single-spa has [different categories](/docs/microfrontends-concept/#types-of-mic
 
 Here is how each single-spa microfrontend works conceptually. This information should help you understand our [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
 
-| Topic                | Application                       | Parcel                               | Utility                              |
-| -------------------- | --------------------------------- | ------------------------------------ | ------------------------------------ |
-| Routing              | has multiple routes               | has no routes                        | has no routes                        |
-| API                  | declarative API                   | imperative API                       | no single-spa API                    |
-| Renders UI           | renders UI                        | renders UI                           | doesn't directly render UI           |
-| Lifecycles           | single-spa managed lifecycles     | custom managed lifecycles            | no lifecycles                        |
-| When to use          | core building block               | only needed with multiple frameworks | useful to share common logic         |
+| Topic                | Application                       | Parcel                               | Utility                                           |
```

Comment:
After tweaking the text in the table the table looked funny in markdown. I increased the length of the column so that it lined up nicely in markdown.